tests/standardized/meta/invalid/invalidAuthors0.json

```diff
@@ -0,0 +1,8 @@
+{
+	"testInfo":{
+		"testCase": "invalid",
+		"errorPointer":"['meta']['authors']",
+		"reason": "'mr. author' is not of type 'array'"
+	},
+	"package":{"manifest":"ethpm/3","meta":{"authors":"mr. author"}}
```

Comment:
@pipermerriam Any thoughts regarding on this structure for the wrapper? From what I read, I couldn't find any kind of "universal" error codes, but it seems like the erroneous json pointer should be capable of that role. It's also unlikely that the `"reason"` the python jsonschema spits out is universal, but it's useful for the python text and to provide some context.

Reply:
I think that it should include:

- validity flag
- JSON-pointers to invalid fields
   - Most cases can probably be captured with a single pointer.
   - There *might* be cases where we need multiple pointers.
   - Classification of the error.  This is probably something you need to make up.  This could be as simple as strings like `invalid-name`.  You can go further and define them using error codes like `N001 - Invalid Contract Type Name`, `N002 - Invalid source file path`.  (The `N` is just an arbitrary letter I choose to mimic the python linting error codes).
    - Maybe a human readable description.

I'd recommend starting simple and we can expand later since for a bit here we'll be the only consumers of these fixtures.